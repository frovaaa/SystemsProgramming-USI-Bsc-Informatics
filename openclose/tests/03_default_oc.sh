#!/bin/bash
exec "$1" <<EOF
The fibonacci() function is inefficient. (Very much so!)
EOF
