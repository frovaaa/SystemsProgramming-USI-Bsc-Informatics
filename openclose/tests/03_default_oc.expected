The fibonacci function is inefficient. (Very much so!)
